### PR TITLE
Add aggregate line item by account

### PR DESCRIPTION
**Pull request by TinyMurky**
### DEVELOP

- add prisma function that aggregate sum(amount) by account id

### CHECK LIST

- [x]  Use `@` in import path
- [Naming convention](https://github.com/CAFECA-IO/WorkGuidelines/blob/main/technology/coding-convention/naming-convention.md) verification: checked
- Coding style verification: checked
- new Library: 0
- new Class / Component: 0
- new loop: 0
- new recursive: 0
- high risk: 0
- new sql: 1
    - lineItem 新增groupBy操作

### UML

- none

### CODE REVIEW

File: src/lib/utils/repo/voucher.repo.ts [hunk 1]
```diff
@@ -207,7 +207,7 @@ export async function getLatestVoucherNoInPrisma(companyId: number) {
       },
     });
 
-    const localToday = `${new Date().getFullYear()}/${new Date().getMonth() + 1}/${new Date().getDate()}`;
+    const localToday = `${new Date().getFullYear()}`.padStart(4, '0') + `${new Date().getMonth() + 1}`.padStart(2, '0') + `${new Date().getDate()}`.padStart(2, '0');
```
Comment (Luphia) [inline src/lib/utils/repo/voucher.repo.ts:210]: - do not use new Date() consecutively.

File: src/lib/utils/repo/voucher.repo.ts [hunk 2]
```diff
@@ -207,7 +207,7 @@ export async function getLatestVoucherNoInPrisma(companyId: number) {
...
     const localTodayStrip = localToday.replace(/\//g, '');
```
Comment (Luphia) [inline src/lib/utils/repo/voucher.repo.ts:211]: - no need for this line

**Review by Luphia — CHANGES_REQUESTED** [review]

File: src/lib/utils/repo/voucher.repo.ts (hunk 1 above)
Comment (TinyMurky) [reply src/lib/utils/repo/voucher.repo.ts]: 修正完成

File: src/lib/utils/repo/voucher.repo.ts (hunk 2 above)
Comment (TinyMurky) [reply src/lib/utils/repo/voucher.repo.ts]: 修正完成

**Review by Luphia — APPROVED** [review]
- no bug found

**Pull request merged**